Calendar/BookingHelpers/BookingStaffLogLogic.ps1

```diff
@@ -0,0 +1,26 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function RunMBStaffLogValidation {
+    [ref]$ErrorMessage = $null
+    [ref]$WriteMessageAlways = $true
+    $TestResult = $true
+
+    Write-DashLineBoxColor "Running Staff Membership Log collection" -Color Blue
+
+    $ErrorMessage = ""
+    $TestResult = CheckBMBStaffMemberShipLog -errorMessage $ErrorMessage
+    WriteTestResult "Get Mailbox Staff membership logs" -success $TestResult -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+}
+
+function CheckBMBStaffMemberShipLog {
```

Comment:
Does this need to be its own function if we only call it once from the function above?

Reply:
As all previous methods on Booking<area>Logic  files, each method is a unit test, more will be added, so each test has to have its own function.

Reply:
How soon till we get pester testing add to this project?